locales/es_ES/pcsx2_Iconized.po

```diff
@@ -607,32 +633,43 @@ msgid ""
 "50% cyclerate. Significant reduction of CPU requirements. Speedup for very "
 "lightweight games, slows down others. FMVs and audio may stutter or skip."
 msgstr ""
+"50% ciclos (tasa). Reducción significativa de requisitos de CPU. Mejora "
+"rendimiento en algunos juegos, empeora en otros. Las cinemáticas y el audio "
+"pueden entrecortarse o ser saltados."
 
 #: pcsx2/gui/Panels/SpeedhacksPanel.cpp:35
 #, c-format
 msgid ""
 "60% cyclerate. Moderate reduction of CPU requirements. Speedup for "
 "lightweight games, slows down others. FMVs and audio may stutter or skip."
 msgstr ""
+"60% ciclos (tasa). Reducción moderada de requisitos de CPU. Mejora "
+"rendimiento en juegos poco exigentes, empeora en otros. Las cinemáticas y el "
+"audio pueden entrecortarse o ser saltados."
 
 #: pcsx2/gui/Panels/SpeedhacksPanel.cpp:41
 #, c-format
 msgid ""
 "75% cyclerate. Slight reduction of CPU requirements. Speedup for less "
 "demanding games, slows down others."
 msgstr ""
+"75% ciclos (tasa). Reducción pequeña de requisitos de CPU. Mejora "
```

Comment:
That phrase could be expressed in a more understandable way, but I thought the 'c' must come after the % or so poedit told me. Is that the case? If it is not I can change it.

Reply:
En Discord me dijeron que podía ignorar los avisos de poedit relacionados con el "%" y "c", te sugiero lo mismo.

Yo lo redactaría de una forma mas natural, ago como...
_75% de la tasa de ciclos. Reduce un poco los requisitos del procesador. Mejora el rendimiento de juegos poco exigentes, ralentiza los demás._

Hay que destacar que hay una progresión en la descripción de los hacks, los porcentajes más bajos solo se recomiendan en juegos MUY, MUY ligeros, los más altos (<100%) para juegos menos ligeros. La traducción debería reflejar ese cambio con términos como estos:

50% **very** lightweight games --> (juegos muy ligeros)
60% lightweight games --> (juegos ligeros)
75% less demanding games --> (_¿juegos poco exigentes?_) o algo así